BTCPayServer/wwwroot/main/site.css

```
@@ -1194,6 +1194,62 @@ input.ts-wrapper.form-control:not(.ts-hidden-accessible,.ts-inline) {
     text-align: right;
 }
 
+/* Column customization */
+#ColumnsDropdownToggle {
+    /* Square icon button whose size equals the standard button height, so it lines up with the sibling filter buttons. */
+    --wallet-columns-btn-size: calc(var(--btcpay-btn-font-size) * var(--btcpay-btn-line-height) + 2 * var(--btcpay-btn-padding-y) + 2 * var(--btcpay-btn-border-width));
+    flex: 0 0 auto;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)